Services/Common/OJS.Workers/OJS.Workers.Executors/StandardProcessExecutor.cs

```diff
@@ -69,7 +69,10 @@ public class StandardProcessExecutor : ProcessExecutor
             var processOutputTask = process.StandardOutput.ReadToEndAsync();
             var errorOutputTask = process.StandardError.ReadToEndAsync();
 
-            await this.WriteInputToProcess(process, inputData);
+            if (inputData is not null)
```

Comment:
why not use `inputData != null`?

Reply:
In recent PRs I've been using ```is not \ is``` as well, decided to start using it since in some places I use ```pattern matching``` ( not in the current PR ).